tests/view_tests/views.py

```diff
@@ -45,7 +46,7 @@ def raises500(request):
     # We need to inspect the HTML generated by the fancy 500 debug view but
     # the test client ignores it, so we send it explicitly.
     try:
-        raise Exception
+        raise Exception("Test exception to trigger debug view")
```

Comment:
```suggestion
        raise Exception
```
This change is not needed